dict/portuguese.yml

```diff
@@ -0,0 +1,291 @@
+
+AMERICA:
+  CARIB:
+    'AG': [Antígua e Barbuda, Antiga e Barbuda, Antiguan*]
+    'AI': [Anguila, Anguilan*, The Valley]
+    'AW': [Aruba, Aruban*, Oranjestad]
+    'BB': [Barbados, Barbadian*, Barbadense*, Bridgetown]
+    'BL': [São Bartolomeu, São-Bartolomense*, Gustávia]
+    'BQ': [Bonaire, Bonairense*, Kralendijk]
+    'BS': [Bahamas, Baamas, Bahamense*, Baamense*, Bahamian*, Baamian*, Bahamês*, Baamês*, Nassau]
+    'CU': [Cuba, Cuban*, Havana]
+    'CW': [Curaçau, Curaçauense*, Willemstad]
+    'DM': [Dominica, Domínica, Dominiquense*, Dominiquês*, Dominicense*, Roseau]
+    'DO': [República Dominicana, Dominican*, São Domingos]
+    'GD': [Granada, Granadin*, São Jorge]
+    'GP': [Guadalupe, Guadalupense*, Basse-Terre]
+    'HT': [Haiti, Haitian*, Porto Príncipe, Porto do Príncipe]
+    'JM': [Jamaica, Jamaican*, Kingston]
+    'KN': [São Cristóvão e Neves, São Cristóvão e Névis, São-Cristovense, Basseterre]
+    'KY': [Ilhas Cayman, Ilhas Caimã, Cayman, Caimão, Caimanês*, Caimanes*]
+    'LC': [Santa Lúcia, Santa-Lucense*, Santa-Luciense*, Castries]
+    'MF': [São Martinho, São-Martinhense*, Marigot]
+    'MQ': [Martinica, Martinican*]
+    'MS': [Monserrate, Monserratense*, Brades, Brades Estate]
+    'PR': [Porto Rico, Porto-Riquenh*, Porto-Riquense*, San Juan, São João]
+    'SX': [São Martinho, São-Martinhense*, Philipsburg]
+    'TC': [Ilhas Turcas e Caicos, Turquense*, Turco-Caicense*, Cockburn Town]
+    'TT': [Trindade e Tobago, Trinidade e Tobago, Trinidad e Tobago, Trinitári*, Trinitin*, Trinitário-Tobagense*, Tobaguian*, Port of Spain, Porto da Espanha, Porto de Espanha]
+    'VC': [São Vicente e Granadinas, São-Vicentin*, Kingstown]
+    'VG': [Ilhas Virgens Britânicas, Virginense*, Road Town]
+    'VI': [Ilhas Virgens Americanas, Ilhas Virgens dos Estados Unidos, Virginense*, Charlotte Amalie, Carlota Amália]
+
+  CENTER:
+    'BZ': [Belize, Belizenh*, Belizense*, Belmopan, Belmopã]
+    'CR': [Costa Rica, Costa-Riquenh*, Costa-Riquense*, Costarriquense*, Costa-Ricense*, Costarricense*, Ticos, San José, São José]
+    'GT': [Guatemala, Guatemaltec*, Guatemalense*, Guatemalês*, Guatemales*, Cidade da Guatemala]
+    'HN': [Honduras, Hondurenh*, Tegucigalpa]
+    'MX': [México, Estados Unidos Mexicanos, Mexican*, Cidade do México]
+    'NI': [Nicarágua, Nicaraguense*, Nica, Manágua]
+    'PA': [Panamá, Panamenh*, Panamense*, Cidade do Panamá]
+    'SV': [El Salvador, Salvadorenh*, San Salvador, São Salvador]
+
+  SOUTH:
+    'AR': [Argentina, Argentin*, Buenos Aires]
+    'BO': [Bolívia, Bolivian*, Sucre, La Paz]
+    'BR': [Brasil, Brasileir*, Brasília, São Paulo, Rio de Janeiro]
+    'CL': [Chile, Chilen*, Santiago, Santiago do Chile]
+    'CO': [Colômbia, Colombian*, Bogotá]
+    'EC': [Equador, Equatorian*, Quito]
+    'FK': [Ilhas Malvinas, Ilhas Falkland, Falklands, Falklander*, Malvinense*]
+    'GF': [Guiana Francesa, Franco-Guianense*]
+    'GY': [Guiana, Guyana, Guianês*, Guianense*]
```

Comment:
Guiana will also match entries for GF...

Reply:
I know this could be confusing, but the problem here is that Guiana is the official name. If you are searching from a Portuguese perspective it is much more likely you will look for the version with "i" instead of the alternative with "y". So I am not sure which would be the proper approach to this case.

@koheiw I am open to your decision on what do you think would be the best way here.

Reply:
This is similar to "Congo". Newspapers writers usually use "French Guiana" for GF explicitly, so "Guiana" in GY is fine (there is no other ways to call it...).